docs/source/tutorials/token_classification/tutorial-p2.ipynb

```diff
@@ -0,0 +1,657 @@
+      "Given label: ORG, suggested label: O\n",
+      "\n",
+      "Sentence 2123 - score=0.000009\n",
+      "\u001b[31mLet\u001b[0m's march together,\" Scalfaro, a northerner himself, said.\n",
+      "Given label: LOC, suggested label: O\n",
+      "\n",
+      "Sentence 2770 - score=0.000010\n",
+      "Scottish \u001b[31mpremier\u001b[0m division after Saturday's matches:\n",
+      "Given label: MISC, suggested label: O\n",
+      "\n",
+      "Sentence 2272 - score=0.000013\n",
+      "\u001b[31mBut\u001b[0m 2 27/11/96 5,000 Burma\n",
+      "Given label: MISC, suggested label: O\n",
+      "\n",
+      "Sentence 604 - score=0.000015\n",
+      "\u001b[31m1.\u001b[0m Fc Cologne 16 8 2 6 31 27 26\n",
+      "Given label: ORG, suggested label: O\n",
+      "\n",
+      "Sentence 2102 - score=0.000017\n",
+      "\u001b[31mNetwork\u001b[0m operators said the draft laws would hold them responsible for copyright infringements in the system and expose them to multi-billion-dollar liabilities.\n",
+      "Given label: MISC, suggested label: O\n",
+      "\n",
+      "Sentence 2273 - score=0.000021\n",
+      "-- \u001b[31mBangkok\u001b[0m newsroom (662) 652-0642\n",
+      "Given label: MISC, suggested label: LOC\n",
+      "\n",
+      "Sentence 3448 - score=0.000022\n",
+      "The lanky former Leeds United defender did not make his England debut until the age of 30 but eventually won 35 caps and was a key member of the \u001b[31m1966\u001b[0m World Cup winning team with his younger brother, Bobby.\n",
+      "Given label: MISC, suggested label: O\n",
+      "\n",
+      "Sentence 1936 - score=0.000023\n",
+      "The basket comprises Algeria's Saharan Blend, Indonesia's Minas, Nigeria's Bonny Light, Saudi Arabia's Arabian Light, \u001b[31mDubai\u001b[0m of the Uae, Venezuela's Tia Juana and Mexico's Isthmus.\n",
+      "Given label: MISC, suggested label: LOC\n",
+      "\n",
+      "Sentence 1106 - score=0.000024\n",
+      "Two ships loaded on the East \u001b[31mCoast\u001b[0m, three waited to load, six were due.\n",
+      "Given label: O, suggested label: LOC\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "scores, token_scores = get_label_quality_scores(labels_nl, pred_probs_nl, return_token_info=True) \n",
+    "ranking = np.argsort(scores) \n",
+    "\n",
+    "for r in ranking[:top]: \n",
+    "    print('Sentence %d - score=%.6f' % (r, scores[r])) \n",
+    "    issue_index = np.argmin(token_scores[r]) \n",
+    "    issue_word = given_words[r][issue_index] \n",
+    "    print(color_sentence(sentences[r], issue_word)) \n",
+    "    given_label = given_labels[r][issue_index] \n",
+    "    suggested_label = np.argmax(pred_probs_nl[r][issue_index]) \n",
+    "    print('Given label: %s, suggested label: %s\\n' % (entities[given_label], entities[suggested_label])) "
```

Comment:
can we make `print_issue()` handle this as well (or introduce a new similar function if truly necessary) so that all this code just becomes one line?

Reply:
I was thinking of making this a collapsed function like the ones to show the plots for MNIST dataset. What do you think?

Reply:
Isn't this useful part of the library though? I'd consider making it a user-facing function inside of the cleanlab.token_classification module if you think it can be put behind a friendly API